src/aluno/aluno.controller.ts

```diff
@@ -13,4 +14,14 @@ export class AlunoController {
     const { id } = request.user;
     return this.alunoService.findByClerkId(id);
   }
+
+  @UseGuards(AuthGuard)
+  @Post(':id/update')
```

Comment:
O middleware `@UseGuards(AuthGuard)` é o responsável por verificar o token enviado pelo frontend, e garantir que o token é válido. A partir disso o usuário se autentica na nossas rotas.

Nessa lógica, quando confirmamos que o token é válido, nós adicionamos via `AuthGuard` tanto o clerk_id quanto a clerk_session dentro da requisição. Isso pode ser visto [aqui](https://github.com/TSIW-PROAE/proae_backend/blob/develop/src/auth/auth.guard.ts#L26-L29).

Com isso, não é necessário utilizar o `:id` no endpoint, apenas o `/update` é o suficente. :)

```suggestion
  @Post('/update')
```